bpaotu/bpaotu/static/bpaotu/bpaotu.js

```diff
@@ -577,10 +577,22 @@ $(document).ready(function() {
             async: true,
             success: function(result) {
                 authentication_token = result;
+
                 run_setup_functions();
+
+                var data_portion = $.parseJSON(authentication_token.split("||")[1]);
+                var username = (data_portion.email).split("@")[0];
```

Comment:
in retrospect I think this'll be confusing -let's just show the full email address

Reply:
Fixed.